frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/SignUpHandler.java

```diff
@@ -186,7 +183,7 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
 
             LOG.info("Successfully processed request");
             try {
-                return generateApiGatewayProxyResponse(200, new SignUpResponse(consentRequired));
+                return generateApiGatewayProxyResponse(200, new SignUpResponse());
```

Comment:
Is the SignUpResponse needed now since it doesn't do anything ?

Reply:
Good point, have removed it